source/core/index-compound.txt

```diff
@@ -110,39 +110,41 @@ following:
 Prefixes
 --------
 
-Compound indexes support queries on any prefix of the index
-fields. Index prefixes are the beginning subset of indexed fields. For
-example, given the index ``{ a: 1, b: 1, c: 1 }``, both ``{ a: 1 }``
-and ``{ a: 1, b: 1 }`` are prefixes of the index.
+Compound indexes support queries on any prefix of the index fields.
+Index prefixes are the beginning subsets of indexed fields. For example,
+``{ a: 1 }`` and ``{ a: 1, b: 1 }`` are both prefixes of the index
+``{ a: 1, b: 1, c: 1 }``.
 
-If you have a collection that has a compound index on ``{ a: 1, b:
-1 }``, as well as an index that consists of the prefix of that index,
-i.e. ``{ a: 1 }``, assuming none of the index has a sparse or unique
-constraints, then you can drop the ``{ a: 1 }`` index. MongoDB will be
-able to use the compound index in all of situations that it would have
-used the ``{ a: 1 }`` index.
+If an index is an exact prefix of a second index, and if neither index has
```

Comment:
I think you moved the wrong paragraph.  That's probably my fault -- I probably commented on an outdated commit, which skewed the line. This needs to go down, so organizationally, we want:

Prefixes
----------
def of prefixes of indexes

examples of queries supported by index

examples of queries not supported by index

Can get rid of other indexes that are prefixes.